package.json

```diff
@@ -23,7 +23,7 @@
     "postinstall": "npm run build",
     "start": "cross-env NODE_ENV=production node --use_strict server/node-server.js",
     "pretest": "npm run clean coverage/ && npm run lint",
-    "test": "nyc --reporter=lcov --reporter=text-summary --all --require babel-register ava",
+    "test": "nyc ava",
```

Comment:
Tests are weird still. I'll try running travis again without cache. This change just moves the nyc configuration around in package.json.